### PR TITLE
Feat/connect to spotify api

### PR DESCRIPTION
**Pull request by razelimel**
Connected to the spotify web api to import artists and tracks to the project.

### CODE REVIEW

File: PartyWebApp/Controllers/PerformersController.cs [hunk 1]
```diff
@@ -54,7 +60,7 @@ public IActionResult Create()
         // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Create([Bind("Id,firstName,lastName,genre,rating,imageUrl")] Performer performer)
+        public async Task<IActionResult> Create([Bind("Id,SpotifyId")] Performer performer)
```
Comment (linoybekker) [inline PartyWebApp/Controllers/PerformersController.cs:63]: Why did you delete the this parameters?

**Review by linoybekker — APPROVED** [review]

File: PartyWebApp/Controllers/PerformersController.cs (hunk 1 above)
Comment (razelimel) [reply PartyWebApp/Controllers/PerformersController.cs:63]: Because I've changed the model and you don't need those parameters anymore, just the SpotifyId.
